### PR TITLE
Emagging a shuttle now violently throws people without seatbelts on ignition, and has a chance to break seatbelts.

### PR DESCRIPTION
**Pull request by Ziiro**
:cl: MrDoomBringer & Ziiro
add: Emagging a shuttle gives the engines extra kick, launching people who are not buckled (50% chance to break seatbelts)
tweak: There is a random message underneath the hijack message now, really letting people know that they're going to want to **buckle the fuck up.**
tweak: All shuttle movement will now move people who aren't buckled 1 tile.
/:cl:

Ports https://github.com/tgstation/tgstation/pull/36050

[why]: I'm pretty sure this will never not be funny.

Additionally, it adds some more risk to the hijacker, and gives them different ways to get people the fuck off their shuttle if they're clever.

dreamseeker_2018-03-01_00-43-36

I can add or remove any lines you find too memey.

### CODE REVIEW

File: code/modules/shuttle/emergency.dm [hunk 1]
```diff
@@ -129,8 +129,9 @@
 		// shuttle timers use 1/10th seconds internally
 		SSshuttle.emergency.setTimer(ENGINES_START_TIME)
 		var/system_error = emagged ? "SYSTEM ERROR:" : null
+		var/emag_boost_message = pick("STRAP IN FOR EMERGENCY BURN", "PREPARE FOR LUDICROUS SPEED", "WHEN THIS BABY HITS 88 MILLION MILES PER HOUR YOU'RE GONNA SEE SOME SERIOUS SHIT", "MAKING THE JUMP TO %@ERROR^& IN UNDER 12 PARSECS", "INERTIAL DAMPENERS OFFLINE", "WHAT IF THE ENGINES MOVE THE UNIVERSE, NOT THE SHUTTLE?")
```
Comment (FlattestGuitar) [inline code/modules/shuttle/emergency.dm]: Parsecs are a unit of distance, not time or speed. 0 / :100:

Comment (Ziiro) [reply code/modules/shuttle/emergency.dm]: 😏

Comment (ike709) [reply code/modules/shuttle/emergency.dm]: I can't tell if you're joking or if you've never seen Star Wars.

Comment (FlattestGuitar) [reply code/modules/shuttle/emergency.dm]: Star Wars is trash.

Comment (SamHPurp) [inline code/modules/shuttle/emergency.dm]: imo remove the back to the future one, and the last one

File: code/modules/shuttle/on_move.dm [hunk 2]
```diff
@@ -324,6 +324,13 @@ All ShuttleMove procs go here
 
 /mob/living/afterShuttleMove(list/movement_force, shuttle_dir, shuttle_preferred_direction, move_dir)
 	. = ..()
+	if(movement_force["IGNOREBUCKLE"] && buckled && prob(50))//GET READY FOR A WILD RIDE
```
Comment (SamHPurp) [inline code/modules/shuttle/on_move.dm]: feels like 50% is too high

File: code/modules/shuttle/emergency.dm (hunk 1 above)
Comment (Ralta) [reply code/modules/shuttle/emergency.dm]: ^ Those are the only two I'd remove as well.

File: code/modules/shuttle/on_move.dm (hunk 2 above)
Comment (Ralta) [reply code/modules/shuttle/on_move.dm]: I think a 50/50 isn't that bad, but maybe drop it to 33%? 1 in 3 seems pretty reasonable.

**Comment by Ziiro** [comment]
Alright. Removed unwanted phrased and changed the chance.

File: code/modules/shuttle/emergency.dm (hunk 1 above)
Comment (ike709) [reply code/modules/shuttle/emergency.dm]: those are literally the best two

**Review by SamHPurp — APPROVED** [review]

**Review by AndrewMontagne — APPROVED** [review]

**Pull request merged**